vllm/worker/model_runner.py

```diff
@@ -168,7 +167,91 @@ def from_broadcasted_tensor_dict(
 
 
 class ModelInputForGPUBuilder(ModelRunnerInputBuilderBase[ModelInputForGPU]):
-    """TBA"""
+    """Build ModelInputForGPU from SequenceGroupMetadata."""
+
+    # Compute function names for each sequence in
+    # a sequence group. The order of the functions matters
+    PER_SEQ_COMPUTE_FN_NAMES = (
```

Comment:
instead of strings, can we just have this as an object attribute and have a list of methods directly by default? that way we can avoid the `getattr`

Reply:
Sure I'll create this list in `__init__` and directly put methods in it. I guess this won't hurt the performance.

Reply:
I think it may even make it better because `getattr` has overhead :)